src/frontend/src/pages/FinancePage/RefundsSection.tsx

```diff
@@ -70,7 +70,9 @@ const Refunds = ({ userReimbursementRequests, allReimbursementRequests }: Refund
   const displayedReimbursementRequests =
     allReimbursementRequests && tabValue === 1 ? allReimbursementRequests : userReimbursementRequests;
 
-  const rows = displayedReimbursements.map(getRefundRowData);
+  const rows = displayedReimbursements.map(getRefundRowData).sort((a, b) => {
```

Comment:
This lambda can be changed to `(a, b) => b.date.valueOf() - a.date.valueOf()` (remove square brackets and `return` keyword)